opm/autodiff/GeoProps.hpp

```diff
@@ -150,7 +134,7 @@ namespace Opm
             // opmfil is hardcoded to be true. i.e the volume weighting is always used
             opmfil = true;
             if (opmfil) {
-                minPvFillProps_(grid, eclState, ntg);
+                minPvFillProps_(grid, eclState, pvol_, ntg);
```

Comment:
I noticed this too, but I wanted to keep the changes as minimal as possible. do you prefer a separate method (e.g., `computePoreVolume()`) or do you want to see the code moved to the `update()` method?